src/main/java/com/powsybl/caseserver/parsers/entsoe/EntsoeFileNameParser.java

```diff
@@ -29,9 +28,11 @@ public class EntsoeFileNameParser implements FileNameParser {
     public static final Pattern FILE_NAME_REGEX = Pattern.compile("(\\d{8}[_]\\d{4})[_](\\w{3})[_](\\w{3}).*");
     public static final String DATE_FORMAT = "yyyyMMdd_HHmm";
 
-    public static DateTime parseDateTime(String dateStr) {
-        DateTimeFormatter df = DateTimeFormat.forPattern(DATE_FORMAT);
-        return df.withZone(DateTimeZone.forID("Europe/Brussels")).parseDateTime(dateStr);
+    public static ZonedDateTime parseDateTime(String dateStr) {
+        DateTimeFormatter formatter = DateTimeFormatter
+                .ofPattern(DATE_FORMAT)
+                .withZone(ZoneId.of("Europe/Brussels"));
```

Comment:
isn't it better to store `Europe/Brussels` in a static final field

Reply:
Indeed! Done